app/views/notifications/index.html.erb

```diff
@@ -20,54 +20,171 @@
 <% if user_signed_in? %>
   <div class="home" id="notifications-container">
     <div class="side-bar">
-      <div class="widget fixed-widget">
-        <header>
-          <h4>notifications</h4>
+      <div class="crayons-card fixed-widget">
+        <header class="crayons-card__header">
+          <h3 class="crayons-card__title">Filter</h3>
         </header>
-        <div class="widget-body">
-          <a class="query-filter-button <%= "selected" if params[:filter].blank? %>" href="/notifications">ALL</a>
-          <a class="query-filter-button <%= "selected" if params[:filter].to_s.casecmp("comments").zero? && params[:org_id].blank? %>" href="/notifications/comments">COMMENTS</a>
-          <a class="query-filter-button <%= "selected" if params[:filter].to_s.casecmp("posts").zero? && params[:org_id].blank? %>" href="/notifications/posts">POSTS</a>
+        <div class="crayons-card__body widget-body">
+          <section class="crayons-card__inner-section">
+            <% if @organizations.present? %>
+              <header class="crayons-card__subheader">
+                <h4 class="crayons-card__subheader__title">
+                  <span>@<%= @user.username %></span>
+                </h4>
+              </header>
+            <% end %>
+            <fieldset class="crayons-fieldset">
+              <a class="query-filter-button <%= "selected" if params[:filter].blank? %>" href="/notifications">
+                <div class="crayons-field crayons-field--radio">
+                  <input type="radio" name="all" id="r2" class="crayons-radio" <%= "checked" if params[:filter].blank? %>>
+                  <label for="all" class="crayons-field__label">
+                    All
+                  </label>
+                </div>
+              </a>
```

Comment:
Having form element with clickable label inside `<a>` is probably a pretty hacky thing to do... What if instead of having a form here AND actual radio buttons we would go with just links and checkmarks on the right like you have here:

![image](https://user-images.githubusercontent.com/108287/79872027-8b894500-83e5-11ea-8acf-15d218938f41.png)

1. It would make much more sense
2. It would be consistent with actual mobile usage

Reply:
Interesting! I thought to use radio options here (even though the existing markup relies on `a` to do the filtering) because from a UX perspective, the radio buttons indicate that these are selectable, mutually exclusive elements.

But you have a good point about this being not semantic. The way I'm using radio options are a bit "artificial", and it's really the `a` that is performing the action. I'm posting this image of the side-by-side of what is the two options we could have. I'll go ahead and remove crayons form elements here so that it's more semantic, and so that it's more specific to notifications filtering menu.

![Screenshot 2020-04-21 14 24 41](https://user-images.githubusercontent.com/816402/79915561-05b4cc00-83dc-11ea-96e1-72a545df1bed.png)

Reply:
👍👍👍

Something to also think about re future iterations of this nav is Reading List where filtering, I believe, does **not** have mutually exclusive options. In fact you can select multiple option. For that reason I think your new design with checkmark would work better since it makes sense as replacement for radio as well as checkboxes (in terms of behaviour concept). 👍